### ISSUE TITLE
Guide To Using I2P In Private Browsing

### ISSUE THREAD

**Issue by Shoalsteed** [eyedeekay/I2P-in-Private-Browsing-Mode-Firefox#125]
create a user guide that:

- explains what extra steps are needed to make use of the full features.
- why containers are great
- Browsing accessing through Legwork or using the I2P Browser container tab
- contains a troubleshooting list 

Explain this and what to do if you cannot access the things in this list: 
<img width="471" alt="Screen Shot 2022-09-28 at 10 39 10 AM" src="https://user-images.githubusercontent.com/50714166/192808762-aad8c627-3983-4a74-9f60-99fa7d7ad9a3.png">

**Comment by Shoalsteed** [comment]
List of topics

**Things that you need to configure**

Legwork ( add to address book
I2pcontrol

**Browser Settings**
- delete as you go
- clear all

**Browsing Options**
- New I2P Browser container tab
- Legwork 

**Router Connections**
-Toopie
- Applications and services

**Troubleshooting**

**FAQ**

**Comment by eyedeekay** [comment]
> Explain this and what to do if you cannot access the things in this list:

But you can, you just have to access the extension homepage first? That's why the applications are on the extension homepage.

**Comment by eyedeekay** [comment]
> explains what extra steps are needed to make use of the full features.

IMO we cross this bridge at another time. They will auto-configure when the plugins are installed, and if all these other much more basic functionality is unclear just punt this down the road for now. They hide themselves from the UI if they don't work, we should focus on the basic functions for now.

**Comment by eyedeekay** [comment]
I do have a fairly substantial wiki: https://github.com/eyedeekay/I2P-in-Private-Browsing-Mode-Firefox/wiki also, it may be useful to harvest information from there

**Comment by eyedeekay** [comment]
Per the implementation of the extension and the site this should go onto the help page from https://github.com/eyedeekay/I2P-in-Private-Browsing-Mode-Firefox/issues/121 which is currently supplied by `i2pcontrol/index.html`, which needs to be moved to `help.html` and broadened to include general help.

**Comment by Shoalsteed** [comment]
_Update Firefox. The only people having this problem are Firefox 91 users who are stuck on an antique version of GNU IceCat. Firefox multi-account containers are automatically activated when you ask permission to use the API in all modern Firefoxes._

- I am using the recent firefox and am unable to access anything in that menu in the screenshot except the I2P Browser. Is there something that I need to do?

**Comment by Shoalsteed** [comment]
> Per the implementation of the extension and the site this should go onto the help page from #121 which is currently supplied by `i2pcontrol/index.html`, which needs to be moved to `help.html` and broadened to include general help.

Yes, was think the same.

**Comment by eyedeekay** [comment]
> _Update Firefox. The only people having this problem are Firefox 91 users who are stuck on an antique version of GNU IceCat. Firefox multi-account containers are automatically activated when you ask permission to use the API in all modern Firefoxes._
> 
>     * I am using the recent firefox and am unable to access anything in that menu in the screenshot except the I2P Browser. Is there something that I need to do?

I don't know what you mean by unable to access anything in that menu. If you can access the `I2P Browser` tab you should be able to access every other tab. Is this the same issue as https://github.com/eyedeekay/I2P-in-Private-Browsing-Mode-Firefox/issues/126#issuecomment-1264003638?

**Comment by Shoalsteed** [comment]
Yes, that was the issue I was talking about. And it seems like we have a workaround established.

**Comment by Shoalsteed** [comment]
I created this to explain the function of containers. Would it make sense to have something like this to show how containers work in non I2P contexts and one with I2P identities to demonstrate tab workflows? 

![containers](https://user-images.githubusercontent.com/50714166/193939684-e0b6b5de-051f-4366-b48e-1097012b8682.png)

**Comment by Shoalsteed** [comment]
![I2PIPB INFO](https://user-images.githubusercontent.com/50714166/196004416-f9ce9a3a-c656-47d4-8a94-af96dd7b9251.png)

**Comment by eyedeekay** [comment]
I would lose the search bars if this is an infographic? They seem confusing.

**Comment by Shoalsteed** [comment]
![guide](https://user-images.githubusercontent.com/50714166/196501359-169c3820-c61a-49c3-8958-1e4cf9b6a24c.png)

**Comment by eyedeekay** [comment]
These ones go with `toopie.html`: https://github.com/eyedeekay/I2P-in-Private-Browsing-Mode-Firefox/tree/master/images
These ones go with the pageAction and browserAction https://github.com/eyedeekay/I2P-in-Private-Browsing-Mode-Firefox/tree/master/icons

- `infotoopie.png`: Shown on all I2P tabs as a pageAction if no other pageAction is in use
- `infotoopies.png`: Shown on I2P tabs as a pageAction if HTTPS is in use on the `.i2p` site
- `infotoopiebt.png`: Shown on I2P tabs if the site contains references to files which are contained in a `.torrent` somwhere
- `infotoopiesbt.png`: Shown on I2P tabs if the site contains references to files which are contained in a `.torrent` somwhere **and** HTTPS is in use on the `.i2p` site

- `i2plogo.png`: used as the main label for I2PIPB, used as the `browserAction` icon and on the extension homepage
- `local_up.png`: exact copy of green light in I2P Java. Shown on proxy indicator pages.
- `torrent.png`: used as a "tab label" in the `browserAction` menu. Click it to tab from browsing control panel to i2psnark-rpc(torrent) control panel in the `browserAction` menu
- `browse.png`: used as a "tab label" in the `browserAction` menu. Click it to tab back to browsing control panel from i2psnark-rpc(torrent) control panel in the `browserAction` menu
